prepare/metrics/llm_as_judge/pairwise_rating/llama_3_ibm_genai_arena_hard_template.py

```diff
@@ -1,6 +1,8 @@
 from unitxt import add_to_catalog
```

Comment:
change file name so its not only ibm_genai

Reply:
done